content/blog/overview-of-the-platform-level-data-model-for-redfish®-device-enablement-standard.md

```diff
@@ -0,0 +1,158 @@
+
+ In addition, the [PLDM base](https://www.dmtf.org/sites/default/files/standards/documents/DSP0240_1.1.0.pdf) specification does not mandate any transport protocol to transfer messages from one component to another. Theoretically, PLDM messages can be transferred by any transport protocol, including the [Management Component Transport Protocol](https://www.dmtf.org/dsp/DSP0239) (MCTP). Hence, a specific document ([DSP0241](https://www.dmtf.org/dsp/DSP0241)) had to be created to define the binding of PLDM messages over MCTP to complete the protocol suite.
+
+## Operational model without PLDM for RDE
+
+Generally speaking, most of today servers are architected around one or more Management Controllers. The majority of the server components are tightly coupled with the MC that has a total control on them in terms of monitoring and configuration. Typical examples are temperature sensors or power supplies. Other more complex components like option cards, also called "external provider devices", are loosely coupled with the MC.
+
+Without the implementation of the PLDM for RDE standard, the MC (iLO) has
+a partial knowledge of external provider device properties and capabilities. This knowledge has been hardcoded in the Redfish service of the iLO firmware. As a consequence, Redfish clients can only get and set the properties referenced in the MC. For an exhaustive management of an external provider device, specific applications like the HPE Smart Storage Administrator (SSA) or the HPE MegaRaid Storage Administrator (MRSA) are needed.
+
+This paradigm (implementation of device properties in the iLO firmware) generates a lock step dependency between the iLO firmware and the device firmware; an update of the device firmware may lead to inaccurate iLO responses or errors if the iLO is not updated with a firmware embedding the new or modified device features.
+
+The picture below shows an iLO based server with two network adapters and one storage adapter. The iLO firmware is in sync with the storage device and one network adapter. This same level of firmware is represented with a green "F". The second network adapter firmware is not in sync with the iLO firmware and, is represented with a red "F". A red cross between the iLO and the un-synced firmware represents potential communication problems.
+
+In addition, this picture mentions the [Bus Master Interface Command](https://data.epo.org/publication-server/document?iDocId=698778&iFormat=0) (BMIC) and the Discovery and Configuration via iLO (DCi) protocols that are respectively patented and HPE proprietary protocols.
```

Comment:
you don't really need the "and" between "patented" and "HPE proprietary"

Reply:
Hummm, I'll keep this "and" separator. There two protocols: the first one is HPE patented, and the second one is just HPE proprietary (not patent).